app/views/lookup_keys/_fields.html.erb

```diff
@@ -51,8 +50,10 @@
       <%= checkbox_f(f, :merge_overrides, :onchange => 'mergeOverridesChanged(this)', :table_field => true,
                         :disabled => !f.object.supports_merge?, :size => "col-md-1", :label_size => "col-md-3",
                         :help_inline => popover("", _("Continue to look for matches after first find (only array/hash type)? Note: merging overrides ignores all matchers that use Puppet default."))) %>
+      <%= checkbox_f(f, :merge_default, :disabled =>  !f.object.merge_overrides, :size => "col-md-1", :table_field => true,
+                        :label_size => "col-md-3", :help_inline => popover('',_("Should the overrides be merged with the default value."))) %>
```

Comment:
The API description's better as it doesn't start with "Should", it simply says what it does ("Include default value..."), I'd say use it here too.